src/app/ApplicationTemplate.Shared.Views/Startup.cs

```diff
@@ -28,6 +29,8 @@ protected override void InitializeViewServices(IHostBuilder hostBuilder)
 		protected override void OnInitialized(IServiceProvider services)
 		{
 			AsyncWebView.Logger = services.GetRequiredService<ILogger<AsyncWebView>>();
+
+			ValidatorOptions.LanguageManager = new FluentValidationLanguageManager();
```

Comment:
I don't know if we can, but could we do this in the CoreStartup rather than the View Startup?

Reply:
done